syntax/testdata/fuzz/FuzzQuote/25f36feab4af00bc4dfc3cf56da02b842b62ba8c5ac44862b5b3b776a0d519b4

```diff
@@ -0,0 +1,3 @@
+go test fuzz v1
```

Comment:
It's not clear for me if these files are really supposed to be here or were just forgotten.

Reply:
Yup they are meant to be left behind: they serve as fuzz seed corpus entries, and ran as tests too.